### PR TITLE
Remove C-ish programming declarations

### PR DESCRIPTION
**Pull request by rafaelcn**
```
Remove the C way of defining functions in C such as: function(void).
```

As the behavior in C++ is completely different. A tiny documentation was
added and more is needed.

Signed-off-by: Rafael Campos Nunes rafaelnunes@engineer.com

### CODE REVIEW

**Comment by rafaelcn** [comment]
What do you think @luizperes?

File: Brain/Brain.cpp [hunk 1]
```diff
@@ -49,7 +51,7 @@ void Brain::run(void)
     int _indexIterations = 0;
 
     if (!_code) {
-        write("No Code!", true);  
+        write("No code given to run", true);
```
Comment (rafaelcn) [inline Brain/Brain.cpp:54]: Does it looks good now @luizperes?

Comment (rafaelcn) [reply Brain/Brain.cpp:54]: Although I think it can be shrink to `No code given`.

Comment (luizperes) [reply Brain/Brain.cpp:54]: For now its ok! :) but I will reduce it even more! ^^

**Comment by luizperes** [comment]
Thanks @rafaelcn LGTM !

**Review by luizperes — APPROVED** [review]

**Pull request merged**